buildtest/utils/file.py

```diff
@@ -143,3 +143,80 @@ def resolve_path(path):
 
     if os.path.exists(real_path):
         return real_path
+
+def read_file(filepath):
+    """ This method provides capability to read a file.
+
+        Parameters:
+
+        :param filepath: file name to read
+        :type filepath: str, required
+        :return: return content of file
+        :rtype: list
+    """
+
+    # type check filepath to ensure its a string, if not return None
+    if not isinstance(filepath,str):
```

Comment:
I think this will need black formatting, should be a space after filepath.

Reply:
that's strange ill fix this, i have pre-commit hook enabled but i run into these issues. Don't worry if you see a failure on black. Ill fix those in subsequent commits if it happens.